src/core_plugins/vega/public/vega_editor.less

```diff
@@ -0,0 +1,49 @@
+@import (reference) "~ui/styles/variables";
+
+.vis-type-vega {
+  .vis-editor-config {
+    // this is the only control, no need for padding
+    padding: 0 !important;
+  }
+
+  // Hiding the "discard" button, which is not needed for a single text entry editor:
+  // * There are other ways to undo: Ctrl+Z, window refresh, or navigating away to another Kibana tab.
+  // * It is dangerous as it may result in large loss of work without a way to undo
+  .navbar-nav li:nth-of-type(3) {
```

Comment:
I think we should add an editorConfig option for this or not hide it, but hiding it with CSS feels like the wrong approach.

Reply:
Yep, this was discussed previously. I will remove it.